spec.md

```diff
@@ -126,6 +127,25 @@ response if the authentication fails.
 Note: Using an authentication mechanism that is agreed to via out of band
 communications could lead to interoperability issues with other platforms.
 
+## URL Properties
+
+This specification defines following properties that might appear within URLs:
+- `service_id`
+- `plan_id`
+- `instance_id`
+- `binding_id`
+- `operation`
+
+While this specification places no restriction on the set of characters used
+within these strings, it is RECOMMENDED that these properties only contain
+characters from the "Unreserved Characters" as defined by
+[RFC3986](https://www.ietf.org/rfc/rfc3986.txt). In other words: uppercase
+and lowercase letters, decimal digits, hypen, period, underscore and tilde.
+
+If a character outside of the "Unreserved Characters" set is used, then
+it SHOULD be percent-encoded prior to be used as part of the HTTP request,
+per [RFC3986](https://www.ietf.org/rfc/rfc3986.txt).
```

Comment:
Is it possible to link directly to the section like this? 
https://tools.ietf.org/html/rfc3986#section-2.1
If that kind of non-pure-text link is inappropriate, can you add to the text direction to see specifically section 2.1?

Reply:
done